src/org/jitsi/impl/neomedia/rtcp/NACKPacket.java

```diff
@@ -237,10 +237,20 @@ public void assemble(DataOutputStream dataoutputstream)
     {
         dataoutputstream.writeByte((byte) (0x80 /* version */ | FMT));
         dataoutputstream.writeByte((byte) RTPFB);
-        dataoutputstream.writeShort(2 + (fci.length / 4));
+        int len = 2 /* sender + source SSRCs */ + (fci.length / 4);
+        if (fci.length % 4 != 0)
+        {
+            len++;
```

Comment:
is it worth renaming it, maybe?  i assumed it was the length in bytes.  `lenInWords` ?